### PR TITLE
Sweeps up some rarely used and unbalanced sources of research points

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Removes the research point gen modes from tesla coils and rad collectors.
Someone is gonna take some flak for removing these at some point, may as well be me.

## Why It's Good For The Game

THE DESIGN DOC DICTACT-

Nah powergen should be valid on it's own, it shouldn't need to rely on other econs to make it worth the work. We can try that later, but not now.

## Changelog
:cl:
del: Removed the research modes from tesla coils and radiation collectors.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Question about the sprite repo,  should the sprites removed here be pr'd there?

**Comment by Fikou** [comment]
add: Added new things
add: Added more things

**Comment by TheVekter** [comment]
@LemonInTheDark You can go ahead and nuke slimes giving research points as well, since I keep meaning to do that and forgetting.

**Comment by Cyprex** [comment]
Why would you REDUCE the amount of methods that generate research points? It already lacks agency, why do this?

**Comment by TheVekter** [comment]
Because all active methods of point generation are going away eventually. Might as well remove the ones we're not using much.

**Comment by Tharcoonvagh** [comment]
what if you removed these stuff AFTER implementing the new research point system, whatever it's like

**Comment by LemonInTheDark** [comment]
> 
> 
> add: Added new things
> add: Added more things

NIGHTCODE MOMENT

**Comment by LemonInTheDark** [comment]
> 
> 
> @LemonInTheDark You can go ahead and nuke slimes giving research points as well, since I keep meaning to do that and forgetting.

Might, might not. My main goal was to grab the ones that'll piss people off so you or whoever finishes research don't have to deal with it.

**Comment by LemonInTheDark** [comment]
> Why would you REDUCE the amount of methods that generate research points? It already lacks agency, why do this?

The current system's goal is to be a time gate, nothing more nothing less. I also don't want things that can be pointed to as a use for power that are just kinda make it something else at pitiful rates.

I understand that this is an interesting feature, but it doesn't fit the current design for techwebs.
> what if you removed these stuff AFTER implementing the new research point system, whatever it's like

See above, this isn't a "IN PREPARATION FOR A FEATURE X 6 YEARS FROM NOW" PR. I'm not building a framework for things in the future, I'm correcting a mistake of the past.

File: code/modules/power/tesla/coil.dm [hunk 1]
```diff
@@ -14,20 +14,13 @@
 	circuit = /obj/item/circuitboard/machine/tesla_coil
 
 	var/zap_flags = ZAP_MOB_DAMAGE | ZAP_OBJ_DAMAGE
-	var/power_loss = 2
```
Comment (ShizCalev) [inline code/modules/power/tesla/coil.dm:17]: Why

**Comment by WarlockD** [comment]
It's fine.  Roundstart rad research is imposable as it is.  Since Tesla arn't bult halfway though the shift there useless too.  I forgot about the slimes.  Don't they also generate cash for the sci budget as well?

File: code/modules/power/tesla/coil.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/power/tesla/coil.dm:17]: it was being used for both versions to decrease the point gen for the research coils. I thought it was redundant, I can put it back if you'd like

**Pull request merged**